docs/Home.md

```diff
@@ -1,58 +1,72 @@
 
-Our main place of communication is on our [Discord Server](https://discord.gg/terasology).
-Make sure to check in, introduce yourself and what you're interested in with regards to Terasology :wave:
-For any playing related issues, leave us a note in the `#play-terasology` channel.
-Troubleshooting workspace setup, compile / test issues or other development related issues can be raised in the `#terasology` channel or by [opening an issue on this repo](https://github.com/MovingBlocks/Terasology/issues/new/choose).
+## Understanding Terasology
 
-Our [forum](https://forum.terasology.org/forum/) is currently mainly used to track progress of our GSoC student projects.
-However, it has a lot of more or less actionable ideas for improvement and a bunch of history of our current gameplays floating around, so feel free to roam around a bit and get inspired :wink:
+Terasology is a mosaic of many components, beautifully combined to create a sandbox for your imagination.
 
-## Contributing
+- 🏛️ **[Architecture Overview](Codebase-Structure.md)**
+  - Understand the high-level structure of the project.
+- 🔧 **[Entity System Architecture](Entity-System-Architecture.md)**
+  - Dive into how our entity system powers the game.
+- 🎲 **[Events and Systems](Events-and-Systems.md)**
+  - Explore how to inject new logic and life into the game.
+- 🏗️ **[Block Architecture](https://github.com/Terasology/TutorialAssetSystem/wiki/Block-Attributes)**
+  - Delve into the building blocks of the game world.
 
-Interested in getting involved with working on the game? Make sure to check out the [Contributor Quick Start](Contributor-Quick-Start.md) for setting up your first workspace and starting the game from source. It also has useful links on how to start with your first contribution.
+![Terasology - High-Level Architecture](architecture.png)
 
-We also apply for GSOC - [Google Summer of Code](https://developers.google.com/open-source/gsoc) and [GCI](GCI.md) - [Google Code-In](https://codein.withgoogle.com/) every year. So if you're a student and it is that time of the year maybe check it out!
 
-## Architecture
+## Quick Links
 
-Terasology is build from many building bricks, that together turn into a game.
+- **[🎮 Start Playing](https://github.com/MovingBlocks/Terasology/blob/develop/docs/Playing.md)**
+- **[🏗️ Developer's Hub](Contributor-Quick-Start.md)**
+- **[🛠️ Troubleshooting](Troubleshooting.md)**
+- **[🛠️ Maintainer's Dashboard](Maintenance.md)**
 
-The _engine_ forms the core, and resides alongside the default _facade_ and _subsystems_ in ([MovingBlocks/Terasology](https://github.com/MovingBlocks/Terasology)).
+## Community and Support
 
-This core is backed by several in-house _libraries_, such as([MovingBlocks/gestalt](https://github.com/MovingBlocks/gestalt)) providing the entity system and module management, or our own UI library ([MovingBlocks/TeraNUI](https://github.com/MovingBlocks/TeraNUI)).
```

Comment:
The information removed here is somewhat important. Are you planning to reintroduce it somewhere else?

Reply:
i we add it back to the home page and figured out the right positioning for it